app/cdap/components/AbstractWidget/StateWrapper.js

```diff
@@ -28,7 +28,7 @@ export default class StateWrapper extends PureComponent {
     widgetProps: PropTypes.object,
     extraConfig: PropTypes.object,
     disabled: PropTypes.bool,
-    errors: PropTypes.object,
+    errors: PropTypes.array,
```

Comment:
I believe this was a bug (found out about it through a console error). We define the errors as `IErrObj[]` in the parent component